### PR TITLE
Portfolio: desktop version

### PR DESCRIPTION
**Pull request by shurichmond**
### Mobile version to Desktop version

- [x] Used media queries in CSS file to obtain desktop version of portfolio.

- [x]  Lay out elements on the page using Grid and Flexbox.

### CODE REVIEW

**Review by levy002 — COMMENTED** [review]
## Hello @shurichmond 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that fails linters. Kindly make sure to fix linter errors and submit again.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @shurichmond   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

- [ ] Kindly make sure your navigation links look as much as possible as the provided Figma design :heavy_check_mark: 

    - ### YOUR DESIGN
    
     image

    - ### FIGMA DESIGN
    image

- [ ] Kindly make sure your form is aligned correctly as much as possible as the provided Figma design. :heavy_check_mark: 

   - ### FIGMA DESIGN
   
    image

   - ### YOUR DESIGN
   
    image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -26,9 +26,9 @@ <h1 class="logo"><a href="#">My Logo</a></h1>
                     fill="#6070FF" />
             </svg>
             <ul class="nav-links">
-                <li class="nav-link"><a href="#">Portfolio</a></li>
-                <li class="nav-link"><a href="#">About</a></li>
-                <li class="nav-link"><a href="#">Contact</a></li>
+                <li class="nav-link">Portfolio</a></li>
+                <li class="nav-link">About</a></li>
+                <li class="nav-link">Contact</a></li>
```
Comment (AlphaNtihinduka) [inline index.html]: - [ ] You have done well to apply the required changes by the previous reviewer on the nav links 👏 👏 . However, you should note that the nav links should be inside `a` tag such that when clicked you are taken to the respective section. Kindly ensure that you apply this to have a professional portfolio ☺️ . Read more here on [geegsforgeegs](https://www.geeksforgeeks.org/how-to-create-links-to-sections-within-the-same-page-in-html/)

File: css/form.css [hunk 2]
```diff
@@ -62,3 +62,22 @@ textarea:focus {
...
+
+@media screen and (min-width: 768px) {
+  .contact-me-form-container {
+    align-content: center;
+    padding: 10% 25%;
+    background-image: url(./assets/contact-form-shapes-desktop.png);
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: contain;
+  }
```
Comment (AlphaNtihinduka) [inline css/form.css]: - [ ] [OPTIONAL] Kindly ensure that the background image used in the Figma design should be visible in your portfolio. Please verify the path used to ensure that your design sticks to the Figma design 👍 .
- [ ] [OPTIONAL] You have done incredibly well to implement the changes. I add on to suggest that you ensure that the `border-radius` used on the inputs of your portfolio is the same as that of the Figma template design.

Your design 😎 
contact

Figma design 
contact fig

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hi @shurichmond 🙋‍♂️
Good job working on implementing the changes required by the previous reviewer. You have done the design to do the expectation and incredibly well. 👏  

TO HIGHLIGHT:
- Your project is professional ✅
- Professional README file :100: 
- Your PR is not yet professional ✅
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

close

## Required changes :recycle: 
 - Kindly consider the inline comments attached ☺️ .

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @shurichmond, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**